ui/pages/settings/developer-options-tab/sentry-test.tsx

```diff
@@ -0,0 +1,185 @@
+import React, { useState, useCallback, ReactElement } from 'react';
+import { ButtonVariant } from '@metamask/snaps-sdk';
+import {
+  Box,
+  Button,
+  Icon,
+  IconName,
+  IconSize,
+  Text,
+} from '../../../components/component-library';
+import {
+  AlignItems,
+  Display,
+  FlexDirection,
+  IconColor,
+  JustifyContent,
+} from '../../../helpers/constants/design-system';
+import { trace } from '../../../../shared/lib/performance';
+
+export function SentryTest() {
+  return (
+    <>
+      <Text className="settings-page__security-tab-sub-header__bold">
+        Sentry
+      </Text>
+      <div className="settings-page__content-padded">
+        <GenerateUIError />
+        <GenerateBackgroundError />
+        <GenerateTrace />
+      </div>
+    </>
+  );
+}
+
+function GenerateUIError() {
+  const handleClick = useCallback(async () => {
+    await window.stateHooks.throwTestError?.('Developer Options');
+  }, []);
+
+  return (
+    <TestButton
+      name="Generate UI Error"
+      description={
+        <>
+          Generate an unhandled <b>TestError</b> in this window.
```

Comment:
We need to include them in message.json to have i18n available for different languages 🙏🏻

Reply:
🤦 Done.